ccl/changefeedccl: TestAlterChangefeedAddTargetsDuringSchemaChangeError failed

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringSchemaChangeError [failed](https://tanzanite.cluster.engflow.com/invocations/default/58771e97-4bd3-4b6e-abe0-a46cb5e5d854?testReportRun=25&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL2NoYW5nZWZlZWRjY2w6Y2hhbmdlZmVlZGNjbF90ZXN0) on release-25.1.0-rc @ [76c2bc942ee1d8d8d62e32047bcf3cacaa21fdc1](https://github.com/cockroachdb/cockroach/commits/76c2bc942ee1d8d8d62e32047bcf3cacaa21fdc1):


```
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516535.935544361,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516535.957074639,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.002241177,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.246788831,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.305314458,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.374892184,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.465668267,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.503179105,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.539807553,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.542272563,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.576863610,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.608610329,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.608729669,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.642384467,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.669186315,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.700716963,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.736550061,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.764348799,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.796436907,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.825329265,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516536.826325355,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 1739516537.256455751,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} NONE 0,0
    alter_changefeed_test.go:1311: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1284: span /Tenant/10/Table/106/{1-2} RESTART 1739516537.256455751,0
    alter_changefeed_test.go:1289: setting boundary timestamp 1739516537.256455751,0
    alter_changefeed_test.go:1331: condition failed to evaluate within 45s: from alter_changefeed_test.go:1338: waiting for checkpoint
    --- FAIL: TestAlterChangefeedAddTargetsDuringSchemaChangeError/cloudstorage (58.06s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=25</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringSchemaChangeError.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-47855

--- a/pkg/ccl/changefeedccl/alter_changefeed_test.go
+++ b/pkg/ccl/changefeedccl/alter_changefeed_test.go
@@ -1198,6 +1198,9 @@ func TestAlterChangefeedAddTargetsDuringSchemaChangeError(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	// Set verbose log to confirm whether or not we hit the same nil row issue as in #140669
+	require.NoError(t, log.SetVModule("kv_feed=2,changefeed_processors=2"))
+
 	rnd, seed := randutil.NewPseudoRand()
 	t.Logf("random seed: %d", seed)
 

